<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addprocess</class>
 <widget class="QWidget" name="addprocess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add process?</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/add.jpg</normaloff>:/res/add.jpg</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加程序</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="processNameEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>请输入程序名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pidEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>请输入pid，仅支持1~999的数字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="timeEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>请输入所需时间，建议1~20之间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="priorityEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>请输入程序优先级，最高优先级为0，最低优先级为255</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="stateBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="currentText">
       <string>已初始化</string>
      </property>
      <item>
       <property name="text">
        <string>已初始化</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>就绪</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>运行中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>阻塞</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>终止</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消 </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
